src/m365/todo/commands/task/task-list.ts

```diff
@@ -0,0 +1,120 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { ToDoTask } from '../../ToDoTask';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  listName?: string;
+  listId?: string;
+}
+
+class ToDoTaskListCommand extends GraphItemsListCommand<ToDoTask> {
+  public get name(): string {
+    return `${commands.TASK_LIST}`;
+  }
+
+  public get description(): string {
+    return 'List tasks from a Microsoft To Do task list';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.listId = typeof args.options.listId !== 'undefined';
+    telemetryProps.listName = typeof args.options.listName !== 'undefined';
+    return telemetryProps;
+  }
+
+  private getListId(args: CommandArgs): Promise<string> {
+    if (args.options.listId) {
+      return Promise.resolve(args.options.listId);
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/me/todo/lists?$filter=displayName eq '${encodeURIComponent(args.options.listName as string)}'`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    return request.get<{ value: [{ id: string }] }>(requestOptions)
+      .then(response => {
+        const listItem: { id: string } | undefined = response.value[0];
```

Comment:
It will be best if we rename this constant `listItem` to something that denotes a list object. `listItem` usually has a special meaning. Do you think it makes sense?